Cargo.toml

```diff
@@ -29,6 +29,8 @@ serde_json = { version = "0.9", optional = true }
 # Disabled until updated to serde 0.9
 # serde_xml = { version = "0.9.1", optional = true }
 
+clippy = { version = ">=0.0, <0.1", optional = true}
```

Comment:
Like with `multipart`, we can probably just go ahead and add this to the `nightly` feature.